### ISSUE TITLE
LOW PRIORITY - Hosted Files, Custom Controls, Custom FPS

### ISSUE THREAD

**Issue by Phantomsane** [44670/desmume-wasm#166]
For the 44670 team,

LOW PRIORITY QUESTIONS - I did look through many closed issues, but I never could find any issue containing these questions, so I hope this isn’t a duplicate!

1. Will you distribute https://ds.44670.org HTML files? I’d like to host this on my small game center (I would just redirect to your site but it’s blocked on school device line) for fellow students that like to play games in their free time but don’t actually have anything to play (and the DS Player is a masterpiece that I like to share with others!).
2. Customizable controls? I didn’t see too many issues about this, but I was wondering in the menu maybe we can drag the controls (snap by custom set amount of pixels, or can be turned off) for people that play on mobile (like me) and want to customize it to their liking especially when playing a specific game that would require customized controls for optimized performance (like in Mario Kart DS, different trigger placement could help improve user’s performance).
3. Custom FPS? Maybe just make it so the time stretches based on how much the FPS is increased (so time doesn’t speed up or slow down too far, ‘cause when I play on my PC that has 165 hz and I disable fps limit it goes super sonic speed lol).

Thanks for your time, 44670 team! Your work truly inspires me and I think that it’s very cool!
Phantomsane

**Comment by Phantomsane** [comment]
I understand your current standpoint in bypassing school blockers. I do understand that this could get your team into some substantial trouble, and potentially stop any work you have going on. I would like to let you know that I am not trying to gaslight you whatsoever, but I would like to have/know how to get the site’s hosted files for personal purposes, such as making my own modifications and possibly committing towards your repository to enhance the experience for others, since I have a team of my own that would be more willing to help. If you have enough time, I’d love a tutorial/explanation on how to get the hosted files to deploy my own version and adjust to optimize various features!
